### PR TITLE
coala 0.8 (new formula)

### PR DESCRIPTION
**Pull request by mr-karan**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

This formula creates a package for https://coala.io

### CODE REVIEW

File: Formula/coala.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+  desc "provides a command-line interface for linting and fixing all your code."
+  homepage "http://coala.io/"
+  url "https://github.com/coala/coala/archive/0.8.1.tar.gz"
+  version "0.8"
+  sha256 "23ece17d2ba0adc147f60b570960a06fe70eef465dead4e1fc95e81e3a8071c1"
+  head "https://github.com/coala/coala.git"
+
+  depends_on :python3
+  resource "appdirs" do
```
Comment (dunn) [inline Formula/coala.rb]: Could you add a blank line above here, and between each resource? Makes reading the formula easier.

File: Formula/coala.rb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    url "https://pypi.python.org/packages/95/59/c70e535f1b3b8eab2279dc58dc5ce1a780eb83efccefa55ca745dc7f02ee/colorlog-2.7.0.tar.gz"
+    sha256 "8e197dae35398049965293021dd69a9db068efe97133597f128e5ef69392f33e"
+  end
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    script = "import coalib"
+    system libexec/"bin/python", "-c", script
```
Comment (dunn) [inline Formula/coala.rb]: Could you make the test a little more involved?  Maybe write out a short bit of code and have coala lint it, like [`shellcheck.rb`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/shellcheck.rb) does.

Comment (mr-karan) [inline Formula/coala.rb]: Hey. This is the package which install `coala` without any bears. I'm going to make another package for `coala-bears` in which such a kind of test will be possible. 

Thanks!

Comment (MikeMcQuaid) [inline Formula/coala.rb]: It sounds like this isn't useful as an application without `coala-bears` so they should be the same package.

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

**Comment by woodruffw** [comment]
Closing due to inactivity.

**Pull request closed without merging**